### DISCUSSION TITLE
URL Protocol for use by the OS protocol handler

### DISCUSSION THREAD

**Discussion by murpii** [Rapptz/discord.py#9374] [Q&A] [answered]
My old bot on discord.py 1.7.3 used to be able to send clickable url link protocols like so by wrapping the url in "<>" symbols like so:
  ```
  player_embed.add_field(name=f"Player: {player_name}",
                              f"\nNon-Steam <ddnet://{address}/>"
                              f"\nSteam: <steam://run/412220//{address}/>",
                              ...
```
This does not seem to work anymore after updating to discord.py 2.2.2. The steam one still works, but the Non-Steam one is just plain text now.

Does anyone know how to fix that?

**Comment by ItsNeil17** [comment]
As far as I know, they removed "ddnet" or whatever its meant to be. You can send clickable URLs through the use of markdown. I'll give you the format below:

`[display text](URL here)` 
An example is: Click [here](https://google.com) to go to google!

Escaped example: Click \[here\]\(https://google.com) to go to google!

**Reply by murpii** [reply]
ddnet was technically never supported. The url just happened to work because it was wrapped in <>. What it basically did is, open the game, connect to a specific game server with given address.

Your example I think requires either http or https to work, which we can't really use here.

**Reply by ItsNeil17** [reply]
Oh, I see, I never used ddnet so I wasn't sure

**Comment by Rapptz** [comment, marked as answer]
The library doesn't do anything with these short of sending them over so whatever changed is entirely on discord's end.

**Reply by murpii** [reply]
Thank you for letting me know. It is unfortunate that the update happened just as I were updating my bot.
